python/sphinx/conf.py

```diff
@@ -0,0 +1,19 @@
+# Copyright: Ankitects Pty Ltd and contributors
+# License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+
+project = 'Anki'
+copyright = '2023, Ankitects Pty Ltd and contributors'
+author = 'Ankitects Pty Ltd and contributors'
+
+REPO_ROOT = os.environ["REPO_ROOT"]
+
+extensions = ['sphinx_rtd_theme', 'autoapi.extension']
+html_theme = 'sphinx_rtd_theme'
+autoapi_python_use_implicit_namespaces = True
+autoapi_dirs = [os.path.join(REPO_ROOT, x) for x in ["pylib/anki", "out/pylib/anki", "qt/aqt", "out/qt/aqt"]]
```

Comment:
I'm getting this error:
```
Extension error:
AutoAPI Directory `/Users/gsgx/code/anki/out/qt/aqt` not found. Please check your `autoapi_dirs` setting.
Traceback (most recent call last):
  File "/Users/gsgx/code/anki/python/sphinx/build.py", line 7, in <module>
    subprocess.run(["out/pyenv/bin/sphinx-build", "out/python/sphinx", "out/python/sphinx/html"], check=True)
  File "/Users/gsgx/code/anki/out/extracted/python/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['out/pyenv/bin/sphinx-build', 'out/python/sphinx', 'out/python/sphinx/html']' returned non-zero exit status 2.
Failed with code Some(1): /Users/gsgx/code/anki/out/pyenv/bin/python python/sphinx/build.py
```

`out/qt/aqt` doesn't exist for me but `out/qt/_aqt` does, is this what you meant to add here?

Reply:
Let's just remove it - by the looks of it, it's empty apart from a tsconfig file.

Reply:
I do see some code in that directory:
```
out/qt/_aqt/forms:
__pycache__      addmodel_qt6.py     browserdisp_qt6.py  clayout_top_qt6.py  editaddon_qt6.py    exporting_qt6.py      findreplace_qt6.py  main_qt6.py         preview_qt6.py     setgroup_qt6.py   synclog_qt6.py
about_qt5.py     addonconf_qt5.py    browseropts_qt5.py  customstudy_qt5.py  editcurrent_qt5.py  fields_qt5.py         forget_qt5.py       modelopts_qt5.py    profiles_qt5.py    setlang_qt5.py    taglimit_qt5.py
about_qt6.py     addonconf_qt6.py    browseropts_qt6.py  customstudy_qt6.py  editcurrent_qt6.py  fields_qt6.py         forget_qt6.py       modelopts_qt6.py    profiles_qt6.py    setlang_qt6.py    taglimit_qt6.py
addcards_qt5.py  addons_qt5.py       changemap_qt5.py    dconf_qt5.py        edithtml_qt5.py     filtered_deck_qt5.py  getaddons_qt5.py    models_qt5.py       progress_qt5.py    stats_qt5.py      template_qt5.py
addcards_qt6.py  addons_qt6.py       changemap_qt6.py    dconf_qt6.py        edithtml_qt6.py     filtered_deck_qt6.py  getaddons_qt6.py    models_qt6.py       progress_qt6.py    stats_qt6.py      template_qt6.py
addfield_qt5.py  browser_qt5.py      changemodel_qt5.py  debug_qt5.py        emptycards_qt5.py   finddupes_qt5.py      importing_qt5.py    preferences_qt5.py  reposition_qt5.py  studydeck_qt5.py  widgets_qt5.py
addfield_qt6.py  browser_qt6.py      changemodel_qt6.py  debug_qt6.py        emptycards_qt6.py   finddupes_qt6.py      importing_qt6.py    preferences_qt6.py  reposition_qt6.py  studydeck_qt6.py  widgets_qt6.py
addmodel_qt5.py  browserdisp_qt5.py  clayout_top_qt5.py  editaddon_qt5.py    exporting_qt5.py    findreplace_qt5.py    main_qt5.py         preview_qt5.py      setgroup_qt5.py    synclog_qt5.py
```

But I'm assuming this isn't relevant for developers, in which case I agree we can just remove this directory

Reply:
The error you're getting is for out/qt/aqt, not out/qt/_aqt. I've updated the path in conf.py - does it work for you now?